src/pages/SellerUpload/ImageUploader.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { useState } from 'react';
+import axios from 'axios';
+
+import styled from 'styled-components';
+
+function ImagePreviewer() {
+  const [selectedImage, setSelectedImage] = useState([]);
+  const [ImageForUpload, setImageForUpload] = useState([]);
+
+  const handleImageAdd = e => {
+    if (e.target.files) {
+      const ImageArray = Array.from(e.target.files).map(file =>
+        URL.createObjectURL(file)
+      );
+      const FileArray = Array.from(e.target.files);
+
+      setImageForUpload(prev => prev.concat(FileArray));
+      setSelectedImage(prev => prev.concat(ImageArray));
+
+      Array.from(e.target.files).map(file => URL.revokeObjectURL(file));
+    }
+  };
+
+  const removeImage = index => {
+    setImageForUpload(param => {
+      let arry = [...param];
+      arry.splice(index, 1);
+      return arry;
+    });
+    setSelectedImage(param => {
+      let arr = [...param];
+      arr.splice(index, 1);
+      return arr;
+    });
```

Comment:
- 이렇게 좀 더 간결하게 작성해볼 수도 있을 것 같습니다!
```suggestion
    setSelectedImage(prev => [...prev].splice(index, 1));
```